### ISSUE TITLE
Improve "complete" example

### ISSUE THREAD

**Issue by dbanksdesign** [amzn/style-dictionary#435]
1. Documentation is very lacking, it looks like someone (me) starting writing some of the READMEs and didn't finish.
1. Get the example to more closely resemble the [demo video](https://www.youtube.com/watch?v=1HREvonfqhY)
1. Add web outputs
1. Add some custom transforms/formats

Or maybe we should rethink having a "complete" example and a "basic" example altogether? Having a "complete" example might make it seem like everyone should start with that one. Just thinking out loud.

**Comment by juanfabrega** [comment]
@dbanksdesign Perhaps this is already included in the above, but an example that generates the docs would be fantastic. Thanks!

**Comment by ocano12** [comment]
@dbanksdesign also an example on how to generate a sketch file. If this exist could you point me in the right direction?

**Comment by dbanksdesign** [comment]
@ocano12 yup that one works. I think we should add a similar example in this repository as well though.

**Comment by avkvirtru** [comment]
If by "web outputs" we mean rendering tokens as HTML presentable somewhere like [Storybook](https://storybook.js.org) or other design system docs, I wholeheartedly agree. 

Design tokens (or a style dictionary) without documentation feels incomplete. A single source of truth is a great start. But how to share and use that truth is paramount. For starters, in our docs, I'm just rendering the raw built files in every format. 

@dbanksdesign curious, what does Amazon do with their style dictionaries?

**Comment by dbanksdesign** [comment]
We output JSON files which then get used in a React or Gastby site as data. I think having a storybook and Gatsby (or other doc site frameworks) would be great

**Comment by kellettemma** [comment]
Hi @dbanksdesign, 

Great documentation! I am working on building Android and iOS tokens for mobile dev team. Will the complete 'android' and 'ios' example work with multiple themes? We are experiencing an issue where the token names need to have unique values across themes. Is there a way of resolving this issue? We have tried Android flavours - is there a similar solution for iOS?

**Comment by oscarb** [comment]
I'll ask here cause I think it's on the same topic but please let me know if you want it as a separate issue.

Correct me if I'm wrong, but in the documentation for the [advanced examples > Component CTI structures > How does it work](https://github.com/amzn/style-dictionary/tree/main/examples/advanced/component-cti#how-does-it-work) the code example doesn't really exemplify what the following paragraph says that it's about...?

![image](https://user-images.githubusercontent.com/17575446/131837896-85c3b9fa-ed2e-4314-ab0b-7c7510c82c04.png)
